README.markdown

```diff
@@ -234,6 +234,8 @@ protoc --plugin=node_modules/ts-proto/protoc-gen-ts_proto ./batching.proto -I.
 
 * With `--ts_proto_opt=returnObservable=true`, the return type of service methods will be `Observable<T>` instead of `Promise<T>`.
 
+* With `--ts_proto_opt=nullPrototypes=true`, a custom prototype will not be used, this will increase memory usage.
```

Comment:
@cliedeman I think this approach looks good, maybe remove this config flag and the extra `integration/simple-nullprototypes`?

Reply:
Its done like this in https://github.com/stephenh/ts-proto/pull/86 unless I misunderstood

Reply:
Yep, I think that commit looks good., but doesn't it mean we can remove this readme change? And the new `simple-nullprotobytes` example? Because now all output will work this way and doesn't need the flag.

Reply:
Oh, sorry, different PR. Yep, I get it.